Events/src/test/java/com/github/swcraftlyon/meetup/infra/dao/MangoEventDaoTest.java

```diff
@@ -0,0 +1,35 @@
+package com.github.swcraftlyon.meetup.infra.dao;
+
+import com.mongodb.client.FindIterable;
+import com.mongodb.client.MongoCollection;
+import com.mongodb.client.MongoDatabase;
+import com.mongodb.client.model.Filters;
+import org.bson.Document;
+import org.junit.Assert;
+import org.junit.Test;
+
+
+public class MangoEventDaoTest {
```

Comment:
Yes there is those kind of prerequisites of course ? you mean that i need to write tests to verify if we have those prerequisites ? I think that we need a embedded version of mongo.